data/navigation_dataset/examples/objects_in_groups/example_0013.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+  <instruction text="the first cone from the right in the row of cones in front of the robot"/>
+  <!-- "file" indicates the which world within certain world type-->
+  <world time="0" file="four_cones/world_0005.xml">
+    <object name="cone0" object_type="cone" object_color="orange" position="-4.53587 -2.50144 0.00962007" orientation="-5.41185e-08 4.60876e-07 -0.00069644 1"/>
+    <object name="cone1" object_type="cone" object_color="orange" position="-4.82003 -1.41204 0.00962007" orientation="-5.40146e-08 4.60793e-07 -0.00069646 1"/>
+    <object name="cone2" object_type="cone" object_color="orange" position="-5.31522 0.375236 0.00962007" orientation="-5.37609e-08 4.61082e-07 -0.00069646 1"/>
+    <object name="cone3" object_type="cone" object_color="orange" position="-5.34313 2.65804 0.00962007" orientation="-5.39365e-08 4.60938e-07 -0.00069644 1"/>
+    <object name="ground" object_type ="ground" object_color="na" position="0 0 0" orientation="0 0 0 1"/>
+    <object name="husky" object_type="robot" object_color="na" position="-0.00114615 0.000733871 0.135478" orientation="-3.09e-06 1.98513e-07 1 0.000961327"/>
+  </world>
+  <NP>
+    <grounding_set>
+      <object name="cone3" object_type="cone" object_color="orange" position="-5.34313 2.65804 0.00962007" orientation="-5.39365e-08 4.60938e-07 -0.00069644 1"/>
+    </grounding_set>
+    <NP>
+      <grounding_set>
+        <object_property object_type="cone" spatial_relation_type="na" index="1"/>
+        <index type="1"/>
```

Comment:
Yes, I think the difference is that this example uses the adjective "first", although I could see how "rightmost" expresses both a spatial_relation and an index. As is, I think this is the convention we used in the manipulation dataset (e.g. example_0129.xml and example_0056.xml), but we can change this.